src/citrine/informatics/workflows/design_workflow.py

```diff
@@ -75,3 +76,77 @@ def executions(self) -> WorkflowExecutionCollection:
         if getattr(self, 'project_id', None) is None:
             raise AttributeError('Cannot initialize execution without project reference!')
         return WorkflowExecutionCollection(self.project_id, self.uid, self.session)
+
+    @property
+    def design_executions(self) -> DesignExecutionCollection:
+        """Return a resource representing all visible executions of this workflow."""
+        if getattr(self, 'project_id', None) is None:
+            raise AttributeError('Cannot initialize execution without project reference!')
+        return DesignExecutionCollection(
+            project_id=self.project_id, session=self.session, workflow_id=self.uid)
+
+
+class NewDesignWorkflow(Resource['NewDesignWorkflow'], Workflow):
```

Comment:
What, if anything, is different between this class and the original DesignWorkflow?